test/renderers_test.js

```diff
@@ -75,6 +75,14 @@ describe("Renderers", function () {
 			.end();
 	});
 
+	it("GET HTML (body)", function () {
+		return tinyhttptest({url: "http://localhost:" + port + "/html?format=text/html"})
+			.expectStatus(200)
+			.expectHeader("content-type", "text/html")
+			.expectBody(/^([^](?!<\/html>))*[^]<\/html>[\n\r\s]*$/gi)
```

Comment:
This hackish regexp checks there is only one ``</html>`` tag on the entire response (which implies the ``</html>`` returned inside JSON is properly escaped).